### ISSUE TITLE
Runtime file is empty?

### ISSUE THREAD

**Issue by iammerrick** [oliviertassinari/serviceworker-webpack-plugin#5]
Is this project ready for use? :-)

**Comment by oliviertassinari** [comment]
@iammerrick Yes, it's.
The [documentation](https://oliviertassinari.github.io/serviceworker-webpack-plugin/) is using it.
And I'm using it in [production](https://splitme.net) (click on the try button to install the sw.js).

**Comment by oliviertassinari** [comment]
@iammerrick Are you experiencing any issue?

**Comment by iammerrick** [comment]
@oliviertassinari Yeah, when I npm install the plugin and require the runtime as shown in the documentation, I get an empty object. If you look at the package on NPM it is publishing an empty file for the runtime.

**Comment by oliviertassinari** [comment]
Yes indeed, this plugin is [hijacking](https://github.com/oliviertassinari/serviceworker-webpack-plugin/blob/master/src/index.js#L55) the require to add his own template, so you don't have to write `sw.js` twice.

**Comment by iammerrick** [comment]
Huh. for whatever reason when I require it I just get an empty module.

**Comment by oliviertassinari** [comment]
Could you `console.log` the two variables here https://github.com/oliviertassinari/serviceworker-webpack-plugin/blob/master/src/index.js#L56?

**Comment by RyanCCollins** [comment]
I am seeing the same issue, which results in the following error when trying to create a new plugin:
```
TypeError: ServiceworkerWebpackPlugin is not a function
    at Object.<anonymous> (/Users/myMac/Developer/senior-web/public-transportation/webpack.config.js:8:23)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at module.exports (/Users/myMac/Developer/senior-web/public-transportation/node_modules/webpack/bin/convert-argv.js:80:13)
    at Object.<anonymous> (/Users/myMac/Developer/senior-web/public-transportation/node_modules/webpack-dev-server/bin/webpack-dev-server.js:55:48)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3TypeError: ServiceworkerWebpackPlugin is not a function
    at Object.<anonymous> (/Users/myMac/Developer/senior-web/public-transportation/webpack.config.js:8:23)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at module.exports (/Users/myMac/Developer/senior-web/public-transportation/node_modules/webpack/bin/convert-argv.js:80:13)
    at Object.<anonymous> (/Users/myMac/Developer/senior-web/public-transportation/node_modules/webpack-dev-server/bin/webpack-dev-server.js:55:48)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3
```

**Comment by sabinmarcu** [comment]
Ran into the same issue just now. 

I ran the same code for the `/docs` and my own app, with `console.logs` at each step (at the script's root, to see if it's loaded, after `apply(compiler)`, after `nmf`, at each `after-resolve` with the variables printed. In the `/docs` example it works just fine, and in my build I only find that I only get the *file loaded* print. 

Will keep looking for a reason (configuration / conflicts)
